docs/faq.md

```diff
@@ -105,7 +105,30 @@ We currently do not have a good way to deduplicate timeout or OOM bugs.
 So, we report only one timeout and only one OOM bug per fuzz target.
 Once that bug is fixed, we will file another one, and so on.
 
-Currently we do not offer ways to change the memory and time limits.
+## Can I change the default timeout and OOM for a fuzz target?
+
+Yes, you can. For this, create a fuzz target options file named `<fuzz-target>.options`,
+where `<fuzz-target>` is the executable file name of the fuzz target, in the same
+directory as your `project.yaml`. The options file can contain fuzzer-specific
+configuration values, such as:
+
+```
+[libfuzzer]
+rss_limit_mb = 6000
```

Comment:
Not sure this is a good example value. The FAQ later says that fuzz targets should not use more than 2.5 GB RAM, but I see at least one project that [bumped the limit](https://github.com/google/oss-fuzz/blob/master/projects/numpy/fuzz_binary_loader.options) to 6 GB.